packages/@yoda/bluetooth/a2dp.js

```diff
@@ -94,6 +94,7 @@ function BluetoothA2dp (deviceName) {
   this._end = false
   this.lastMode = protocol.A2DP_MODE.SINK
   this.localName = deviceName
+  this.stateHit = false

```

Comment:
What's the field for?

Reply:
And I think we have some short description for the above 3 states.

Reply:
Add simple description.